content/es/docs/concepts/workloads/pods/podpreset.md

```diff
@@ -0,0 +1,95 @@
+---
+reviewers:
+- jessfraz
```

Comment:
Is @jessfraz the right reviewer for page localized to Spanish?

Reply:
It probably comes from the English version.
```suggestion
- raelga
```